ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Generators.hs

```diff
@@ -153,6 +158,45 @@ instance ShelleyBasedEra era
       => Arbitrary (SomeSecond (NestedCtxt f) (ShelleyBlock era)) where
   arbitrary = return (SomeSecond indexIsTrivial)
 
+{-------------------------------------------------------------------------------
+  Generators for shelley ledger config
+-------------------------------------------------------------------------------}
+
+instance ( Mock (Crypto era)
```

Comment:
Would you add a Haddock comment similar to that on `arbitraryGlobalsWithFixedEpochInfo` or referencing it, eg. Thanks.